integration_tests/src/main/python/data_gen.py

```diff
@@ -160,15 +160,15 @@ def with_special_pattern(self, pattern, flags=0, charset=sre_yield.CHARSET, weig
             length = int(len(strs))
         except OverflowError:
             length = _MAX_CHOICES
-        return self.with_special_case(lambda rand : strs[rand.randint(0, length)], weight=weight)
+        return self.with_special_case(lambda rand : strs[rand.randrange(0, length)], weight=weight)
```

Comment:
I found this during debugging. `randrange` is not inclusive on the high end, but `randint` is